Documentation/network/servicemesh/envoy-circuit-breaker.rst

```diff
@@ -0,0 +1,251 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+.. _gs_envoy_circuit_breaker:
+
+*******************
+L7 Circuit Breaking
+*******************
+
+Cilium Service Mesh defines a ``CiliumClusterwideEnvoyConfig`` CRD which allows users
+to set the configuration of the Envoy component built into Cilium agents.
+
+Circuit Breaking is an important pattern for creating resilient microservice applications. 
```

Comment:
I don't think Circuit Breaking is a proper noun needing to be capitalized here

Reply:
Istio is written like this, for example https://istio.io/latest/docs/tasks/traffic-management/circuit-breaking/ , this is more in line with everyone's habits, what do you think?  
Do you want to rewrite it this way?
`Circuit Breaking` -> `Circuit breaking`?

Reply:
Hey @tanjunchen let's go ahead and use `Circuit breaking`. Can you make the change and then we can mark this as resolved?